iOS/ViewControllers/Courses/CertificatesListViewController.swift

```diff
@@ -60,20 +105,25 @@ extension CertificatesListViewController { // TableViewDelegate
         self.navigationController?.pushViewController(pdfViewController, animated: true)
     }
 
-    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "certificateCell", for: indexPath)
-        let certificate = self.certificates[indexPath.section]
-        cell.textLabel?.text = certificate.name
-        cell.textLabel?.backgroundColor = .white
-        cell.detailTextLabel?.text = self.stateOfCertificate(withURL: certificate.url)
-        cell.detailTextLabel?.backgroundColor = .white
-        cell.enable(certificate.url != nil)
-        cell.accessoryType = certificate.url != nil ? .disclosureIndicator : .none
-        return cell
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        if #available(iOS 11.0, *) {
+            self.navigationItem.hidesSearchBarWhenScrolling = false
+        }
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if #available(iOS 11.0, *) {
+            self.navigationItem.hidesSearchBarWhenScrolling = true
+        }
```

Comment:
`viewWillAppear` and `viewDidAppear` can be remove since we don't have a search for certificates